src/index.html

```diff
@@ -8,6 +8,11 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="../node_modules/xel/stylesheets/material.theme.css">
   <script src="../node_modules/xel/xel.min.js"></script>
+
+  <link href="../node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
+  <script src="../node_modules/jquery/dist/jquery.js"></script>
+  <script src="../node_modules/bootstrap/dist/js/bootstrap.min.js"></script>
+<!-- bootstrap --> 
```

Comment:
I'm getting this error my dude. If you can fix this somehow then keeping the bootstrap/jquery is fine, but if not then you should just remove it since we're not currently using it.
<img width="262" alt="error1" src="https://user-images.githubusercontent.com/9039337/31478843-360642e4-aee1-11e7-82cf-1976a34f6af3.png">